### ISSUE TITLE
Community Chart Suggestion: Nextcloud All-In-One

### ISSUE THREAD

**Issue by pabloeisenhut** [truenas/apps#450]
https://github.com/nextcloud/all-in-one

**Comment by orrinwitt** [comment]
YES PLEEEEEASE!!! Getting Nextcloud AIO running successfully is the last thing I need to test before migrating my main server over to scale from core, and having this very popular app in the catalog would be a godsend!

**Comment by stavros-k** [comment]
There are no plan currently to add this, considering there is already a nextcloud app.
That being said, you can use the custom option to install it yourself.

**Comment by stavros-k** [comment]
There isn't an ETA I can give you, but are not declined!

**Comment by stavros-k** [comment]
Talk setup issue is still open and will be done eventually (it will be part of the new nextcloud app, currently on test train, feel free to test and send feedback https://forums.truenas.com/t/nextcloud-test-train-feedback-request/37296), but AIO won't be added. 
I don't think there is any reason to keep requests open for something that won't be done. Closing this now.

Thanks

**Comment by orrinwitt** [comment]
For those who find this thread hoping to see Nextcloud AIO added to the official truenas app catalog, before you leave disappointed, here's a working yaml file you can use to spin it up as a custom app. I've been using this for some time now, and it's rock solid. 
(obviously, you will need to set up your corresponding datasets, but I'm assuming you know that already)
I upped the memory limit, but you can put it back to 4gb if needed.


`networks: {}
services:
  nextcloud:
    container_name: nextcloud-aio-mastercontainer
    environment:
      - APACHE_PORT=11000
      - APACHE_IP_BINDING=0.0.0.0
      - NEXTCLOUD_DATADIR=/mnt/tank/nextcloud_data/
      - NEXTCLOUD_MEMORY_LIMIT=8192M
      - NEXTCLOUD_MAX_TIME=36000
      - SKIP_DOMAIN_VALIDATION=true
    image: nextcloud/all-in-one:latest
    ports:
      - '8080:8080'
    restart: always
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
      - /var/run/docker.sock:/var/run/docker.sock:ro
volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer
`
